backend/company_api.py

```
@@ -0,0 +1,776 @@
+            )
+            if not any(ds.name == system.name for ds in existing_systems):
+                await store.create_detected_system(system)
+                log.debug(f"Added detected system {system.name} ({system.system_type}) to company {company_id}")
+    
+    return result
+
+
+@router.post("/{company_id}/scan/repo", response_model=WebsiteScanResult)
+async def scan_repo_endpoint(
+    company_id: str = Path(..., description="Company ID"),
+    request: RepoScanRequest = Body(...),
```

Ruff (0.15.14)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)